detect_numa

```diff
@@ -26,11 +26,21 @@ usage()
     exit 0
 }
 
+# Verifies that NUMA exists on the system
+has_numa()
+{
+    local numa_nodes=`lscpu | grep NUMA | grep CPU | wc -l`
+    if [ -z "$numa_nodes" ] || [ $numa_nodes -le 0 ]; then
+        echo 0
+    fi
+    echo 1
```

Comment:
you should have an else here, we will be echoing both a 0 and a 1.

Reply:
Fixed